examples/vision/python_super_resolution/License

```diff
@@ -0,0 +1 @@
+Copyright (C) 2021 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
What license is this? Why is this different than the license [here](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/LICENSE)?

Reply:
I had a meeting with Legal.  After showing what it looks like and the top level MIT License they walked back their request to have a license specific to the 2 jpg files.  I have added a commit remove these licenses.